spec/support/sanitize.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+def sanitize(html)
```

Comment:
1/ should we add a test for this file @junan ?

```
it 'creates sanitize file' do
    expect(file('sanitize.rb')).to exist
  end
```

2/ Should we consider apply this file for only web variants?